### PR TITLE
GPE-2971 replace travis CI with GitHub actions

### PR DESCRIPTION
**Pull request by matstrand**

### CODE REVIEW

File: .github/workflows/pypi-publish.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: mstrand
```
Comment (matstrand) [inline .github/workflows/pypi-publish.yml:28]: @scriptsrc has this as `__token__` in yamale
https://github.com/23andMe/Yamale/pull/161/files#diff-29239970d55958739fb39189ad2b2b5afe4184fc541ccaa9ee3f91be06877f0fR28

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: Python package
```
Comment (matstrand) [inline .github/workflows/run-tests.yml]: this seems to be working 
https://github.com/23andMe/seqseek/actions

File: tox.ini [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
-    py27: python2.7
-    py3: python3
+
+[gh-actions]
+envlist = py27, py3
+python =
+    3.6: py36
+    3.7: py37
+    3.8: py38
+    3.9: py39
```
Comment (patrickyee23) [inline tox.ini:15]: do we need a py39 env defined?

File: tox.ini [hunk 4]
```diff
@@ -1,11 +1,18 @@
...
 commands = py.test --cov seqseek --cov-report term-missing seqseek
 deps =
   pytest
   pytest-cov
-basepython =
-    py27: python2.7
-    py3: python3
+
+[gh-actions]
+envlist = py27, py3
```
Comment (patrickyee23) [inline tox.ini]: i guess this is not used?

File: .github/workflows/pypi-publish.yml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
```
Comment (patrickyee23) [inline .github/workflows/pypi-publish.yml:25]: just curious about when should we run with `python -m pip ...`. not a big deal.

File: tox.ini (hunk 3 above)
Comment (matstrand) [reply tox.ini:15]: actually i don't think we need any of these defined. I've reduced the envlist to py3 and these all seem to run in github actions

https://github.com/23andMe/seqseek/actions/runs/1280644278

File: tox.ini (hunk 4 above)
Comment (matstrand) [reply tox.ini]: good point, deleted

File: .github/workflows/pypi-publish.yml (hunk 5 above)
Comment (matstrand) [reply .github/workflows/pypi-publish.yml:25]: for this i copied it from the github docs 
https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#publishing-to-package-registries

Comment (matstrand) [reply .github/workflows/pypi-publish.yml:25]: for this i copied it from the github docs 
https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#publishing-to-package-registries

**Review by patrickyee23 — APPROVED** [review]

**Review by mlab23 — APPROVED** [review]

**Pull request merged**